spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
- [ ] I can see that you might have forgotten to write unit and integration tests for your app. Please install the RSpec configs and do so. 😄

Reply:
Dear @MahmoudBakr23 , sorry, we thought it wasn't required.